apps/web/src/pages/dao/[network]/[token]/[tokenId].tsx

```diff
@@ -111,11 +112,19 @@ const TokenPage: NextPageWithLayout<TokenPageProps> = ({
       : baseSections
   }, [hasThreshold, collection])
 
-  // remove line breaks and formatting from og description
-  const cleanDesc = description.replace(/(\r\n|\n|\r|\t|\v|\f|\\n)/gm, '')
+  const ogDescription = useMemo(() => {
+    if (!description) return ''
+    const isMarkdown = isPossibleMarkdown(description)
 
-  const ogDescription =
-    cleanDesc.length > 111 ? `${cleanDesc.slice(0, 111)}...` : cleanDesc
+    if (isMarkdown) {
+      return `${
+        name || 'This DAO'
+      } was created on Builder Nouns. Please click the link to see more.`
```

Comment:
replace `Builder Nouns` with `Nouns Builder`. maybe ping @isaaczaak for a quick double check on the copy

Reply:
yes, `Nouns Builder` is the right copy @jordanlesich @neokry